Add "Manage Following" Page

infinite-persistence opened the pull request:
## Ticket
- Indirectly closes #647
- Closes #986

## Test
`kp` dev instance

tzarebczan commented:
Good start! 

Both buttons now using up too much space on mobile view on following page? 

I didn't see followers loaded (unless you've been to the channel). Not sure if bug, or just due to my large list. When  i refreshed, it worked. Then scrolling down, new ones didn't load. I've seen this on and off in other areas, so maybe a deeper and random bug? 

3 button menu was hard to click for me, but then started working after (mobile).

On manage page, followed channels text + box + search icon (padding) need alignment.

![Screenshot_20220228-082259](https://user-images.githubusercontent.com/8120721/155990574-7724430f-5b4f-4545-afe5-6f12783cb387.png)

![Screenshot_20220228-082900](https://user-images.githubusercontent.com/8120721/155991363-ca986518-3b30-4f3b-b623-3fff28c4ef57.png)

infinite-persistence commented:
[c96fa9a ](https://github.com/OdyseeTeam/odysee-frontend/commit/c96fa9a10d5c23dad98d9ac6a84e5db432ff0f5c)to [a26a602 ](https://github.com/OdyseeTeam/odysee-frontend/commit/a26a60237e5f07c82276689180b22aff8ac64e2a)
- Fixed sidebar resolving entire subs list.
    - Wasn't able to replicate the unresponsive 3-dot issue, but I think it's just due to the sidebar resolving all your subs.  I didn't notice this previous because it was being done outside of the component.
- Moved away from the `claim_search` approach and just manually batch-resolve instead.
    - Too many issues:  slow, some results duplicated in multiple pages, some results missing, full results doesn't match given channel IDs, etc.
- Fixed padding issues.
